scripts/appbuilder

```diff
@@ -58,9 +56,16 @@ done
 
 ### defaults
 
-# if no options specified, assume first argument is an image name
-shift $((OPTIND-1))
-IMAGE_NAME=${IMAGE_NAME:-$@}
+if [ $# -lt 1 ]; then
+        echo "Dockerfile or ImageName has not been specified"
```

Comment:
Does this mean we can't run `appbuilder` byitself anymore?

Reply:
Yes. And this situation(run appbuilder by itself) does not happen to any tests or samples in current repo.